src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -130,7 +141,35 @@ const MapOverlays = ({
                 key={poi.id}
                 location={poi.location}
                 text={poi.displayName}
-                onPress={() => {}}
+                onPress={() => {
+                  setMarkerLocation(poi);
+                }}
+              />
+            ))}
+        </>
+      ) : null}
+
+      {startTravelPlan ? (
+        <>
+          {getAllPOI()
+            .filter(marker => {
+              if (startTravelPlan) {
```

Comment:
This extra check for startTravelPlan is not necessary

Reply:
it is since we are rendering the markers at any zoom level. it would also not be recommended to render the marker twice

Reply:
But you already do this check on line 152, no? This block is only ever activated when startTravelPlan is true. Why do a second check on line 156?

Reply:
ahh right, I didn't see it in github comments